cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,44 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+=====================
+Application Logbacks
+=====================
+
+YARN containers spawned by a CDAP application use the default container logback, `logback-container.xml`,
```

Comment:
Have you been writing Markdown? Two back-ticks (fore and aft) are needed for a literal in reST.